lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -141,7 +143,7 @@ export class PublicClientApplication {
             } else {
                 BrowserUtils.navigateWindow(loginRequestUrl, true);
```

Comment:
Will this trigger a refresh if you are already on the loginRequestUrl?

Reply:
Thanks for the callout. I have added a check for this the same way as in #1358.

Reply:
We also want to make sure the hash is processed if the currentUrl is the loginRequestUrl. Currently will return null.